docs/content/v2024.1/benchmark/sysbench-ysql.md

```diff
@@ -37,55 +67,63 @@ Make sure you have the [YSQL shell](../../api/ysqlsh/) `ysqlsh` exported to the
 $ export PATH=$PATH:/path/to/ysqlsh
 ```
 
-### Start YugabyteDB
+  </div>
 
-Start your YugabyteDB cluster by following the steps in [Manual deployment](../../deploy/manual-deployment/).
+  <div id="rhel" class="tab-pane fade" role="tabpanel" aria-labelledby="rhel-tab">
 
-{{< tip title="Tip" >}}
-You will need the IP addresses of the nodes in the cluster for the next step.
-{{< /tip>}}
+```sh
+wget https://github.com/yugabyte/sysbench/releases/download/1.0.0-yb/sysbench-1.0.0-1.el8.x86_64.rpm
 
-### Run the benchmark
+sudo yum install -y sysbench-1.0.0-1.el8.x86_64.rpm 
+```
 
```

Comment:
where is sysbench installed?

Reply:
sysbench gets installed as a cli binary and is accessible from any path(as after installation it is in /usr/local/bin/)